Dockerfile-assemblyscript

```diff
@@ -0,0 +1,16 @@
+FROM node:latest
+
+WORKDIR /tmp
+
+# NOTE: I have presumed we'd pass these env vars via the Docker -e flag.
```

Comment:
Actually I'm not sure what I was thinking here, because it doesn't look like goreleaser let's me configure any form of docker related flags and so goreleaser is just handling things internally as far as building a docker image is concerned.